### ISSUE TITLE
about thatgamecompany

### ISSUE THREAD

**Issue by tkdlxjtm** [caffeine-overload/bandinchina#176]
more descreption is here[#](https://twitter.com/Sky_harmonia/status/1357357569266778113?s=19)

**Comment by WAUthethird** [comment]
I disagree that this situation has _anything_ to do with censorship. I don't think the actions of Jenova Chen are enough to put his company on this list. It seems like it was a cultural controversy, not one of blocking fundamental access to knowledge.

That is, unless I missed something? I can't read Chinese/Korean, (which many of the comments were) so that's probably likely.

**Comment by Battlecruiser0** [comment]
![Screenshot_20210206-190337_NAVER](https://user-images.githubusercontent.com/78641147/107115165-102e4c80-68ae-11eb-839c-3c41d12d8537.jpg)
![NSC20210206_190307](https://user-images.githubusercontent.com/78641147/107115168-158b9700-68ae-11eb-8ada-96ddd8f78271.jpg)
It seems like there is a suspicious act of reply deleting.

**Comment by caffeine-overload** [comment]
From what I've seen so far I'd tend to agree with @WAUthethird

**Comment by tkdlxjtm** [comment]
@Battlecruiser0 sorry to realize that. i guess that reply deleted because that user is deleted accout.
https://pbs.twimg.com/media/EtnQZ-wVEAIB801?format=jpg&name=medium <here's another image for info

**Comment by Battlecruiser0** [comment]
@tkdlxjtm Well, my explanation was not enough. I think the effective censorship is one without been noticed to public. What I am doubting is about the long patterns of account deletation by something that observes this issue. I know that it wouldn't make sense to someone, but I believe dubious attitude will fend against the CCP.

**Comment by DevilNANA** [comment]
https://namu.wiki/w/%EC%A4%91%EA%B5%AD%EC%9D%98%20%ED%95%9C%EA%B5%AD%20%EB%AC%B8%ED%99%94%20%EC%98%88%EC%86%8D%ED%99%94%20%EC%8B%9C%EB%8F%84
https://namu.wiki/w/Sky%20-%20%EB%B9%9B%EC%9D%98%20%EC%95%84%EC%9D%B4%EB%93%A4
In South Korea, that think this way. Joined in China's attempt to subjugate Korean culture.

**Comment by vus7294** [comment]
#177
